.github/workflows/crowdin-upload.yml

```
@@ -0,0 +1,58 @@
+name: Crowdin Upload
+on:
+  workflow_dispatch:
+  schedule:
+    # runs every day at 11:00 AM UTC
+    - cron: "0 11 * * *"
```

```suggestion
  # schedule:
    # runs every day at 11:00 AM UTC
    # - cron: "0 11 * * *"
```
comment it out so we just have workflow_dispatch for manual runs